core/src/main/scala/shapeless/generic.scala

```diff
@@ -288,6 +288,13 @@ trait CaseClassMacros extends ReprTypes with CaseClassMacrosVersionSpecifics {
     normalized <:< hlistTpe || normalized <:< coproductTpe
   }
 
+  /**
+   * Lower the order of type kind applying Any type.
```

Comment:
Now I notice that it works only for types with one type parameter, so maybe we should document that.

Reply:
thanks a lot!

Reply:
@joroKr21 

```scala
import scala.reflect.runtime.universe._

def lowerKind(tpe: Type): Type =
  if(tpe.takesTypeArgs)
    appliedType(tpe, List(typeOf[Any])).dealias
  else tpe

lowerKind(typeOf[List[_]].typeConstructor)
lowerKind(typeOf[Map[_, _]].typeConstructor)
// val res0: reflect.runtime.universe.Type = List[Any]
// val res1: reflect.runtime.universe.Type = Map[Any]
```

I confirmed lowerKind behaviors locally, and is this result an ideal one?
Map is applied one type parameter, but this looks strange.

Reply:
> Map is applied one type parameter, but this looks strange.

I have no clue what that means tbh

Reply:
Sorry, I misunderstood the work of `appliedType`.
I didn't know `appliedType` can be used for any type like below.

```scala
appliedType(typeOf[String], typeOf[Any])
// val res0: Type = String[Any]
```

So, I thought `Map` should take 2 type parameters and Map with one type parameter looked strange to me.
I figure out how `appliedWorks` and the reason of `if(tpe.takesTypeArgs)` exists.

Reply:
I added the comment about this topic, but feel free to say any point to be improved.

Reply:
The other option would be to fix it instead:
```scala
def lowerKind(tpe: Type): Type =
    appliedType(tpe, tpe.typeArgs.map(_ => AnyTpe)).dealias
```